backend/src/app.ts

```diff
@@ -48,4 +51,31 @@ app.get('/reviews/:idType/:id', async (req, res) => {
   res.status(200).send(JSON.stringify(reviews));
 });
 
+app.post('/landlords', async (req, res) => {
```

Comment:
It is probably a good idea to try/catch all of these handlers, so that it won't crash the function or return a 201 when something goes wrong